test/runtests.jl

```diff
@@ -335,7 +335,18 @@ end
         @test isapprox(dotc(xc, yc) , dotc(convert(Gray, xc), convert(Gray, yc)), atol=1e-6)
         @test dotc(RGB(1,0,0), RGB(0,1,1)) == 0
     end
-end
 
+    @testset "typemin/max" begin
+        for T in (Normed{UInt8,8}, Normed{UInt8,6}, Normed{UInt16,16}, Normed{UInt16,14}, Float32, Float64)
+            @test typemin(Gray{T}) == typemin(T)
```

Comment:
Might be best to test with `===`, e.g., `typemin(Gray{T}) === Gray{T}(typemin(T))` or something. Otherwise I don't think we're testing that `typemin(T)` returns a value of type `T`.